recipes/ukcensusapi/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ukcensusapi" %}
+{% set version = "1.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/a4/88/e13b9136723fb4617041904243cd24ccddcff5324868c963ab72564d04bc/{{ name }}-{{ version }}.tar.gz
```

Comment:
Can you use the canonical url: 

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14